chaco/tools/simple_zoom.py

```diff
@@ -38,13 +38,23 @@ class SimpleZoom(AbstractOverlay, ToolHistoryMixin, BaseZoomTool):
     #:   Perform a "box" selection on two axes.
     tool_mode = Enum("box", "range")
 
+<<<<<<< HEAD
+    # Is the tool always "on"? If True, left-clicking always initiates
+    # a zoom operation; if False, the user must press a key to enter zoom mode.
+    always_on = Bool(False, transient=True)
+
+    # Defines a meta-key, that works with always_on to set the zoom mode. This
+    # is useful when the zoom tool is used in conjunction with the pan tool.
+    always_on_modifier = Enum(None, "shift", "control", "alt", transient=True)
+=======
```

Comment:
looks like a merge conflict crept in - and the chaco testsuite sadly didnt capture this!